frontend/app/tests/e2e/pages/rotki-app.ts

```diff
@@ -190,7 +190,7 @@ export class RotkiApp {
 
   static navigateTo(menu: string, submenu?: string) {
     const click = (selector: string) => {
-      cy.get(selector).scrollIntoView();
+      cy.get(selector).scrollIntoView({ offset: { top: -200, left: 0 } });
```

Comment:
is there a problem with scroll? is there another way to resolve it?

Reply:
the expansion transition (height 0 => full) messing up the scrolling. let me think other way.